content/modules/ROOT/pages/module-02.adoc

```diff
@@ -173,9 +177,11 @@ image::02/02-open-terminal.png[]
 +
 [source,bash,role="execute",subs="+macros,+attributes"]
 ----
-curl -X POST -H "Content-Type: application/json" -d '{"model":"ibm-granite/granite-3.2-8b-instruct" ,"prompt": "Hello, how are you?"}' http://granite-3dot2-8b-instruct-predictor.llm-hosting.svc.cluster.local:8080/v1/completions
+curl -X POST -H "Content-Type: application/json" -d '{"model":"ibm-granite/granite-3.2-8b-instruct" ,"prompt": "Hello, how are you?"}' http://granite-3dot2-8b-instruct-predictor.llm-hosting.svc.cluster.local:8080/v1/completions | jq
 ----
 
+_When pasting content for the first time, your workbench may prompt you to grant paste access. Please allow it to enable copy-paste functionality._
```

Comment:
I suggest adding this as an admonition instead of just an italicized line in the middle of the steps.

Reply:
Done. Added a NOTE section it renders much better on screen.